pelican/urlwrappers.py

```diff
@@ -115,10 +116,11 @@ class Category(URLWrapper):
     @property
     def slug(self):
         if self._slug is None:
-            substitutions = self.settings.get('SLUG_SUBSTITUTIONS', ())
-            substitutions += tuple(self.settings.get('CATEGORY_SUBSTITUTIONS',
-                                                     ()))
-            self._slug = slugify(self.name, substitutions)
+            if 'CATEGORY_REGEX_SUBSTITUTIONS' in self.settings:
+                subs = self.settings['CATEGORY_REGEX_SUBSTITUTIONS']
+            else:
+                subs = self.settings.get('SLUG_REGEX_SUBSTITUTIONS', [])
+            self._slug = slugify(self.name, regex_subs=subs)
```

Comment:
There is a behavior change here. Before, `CATEGORY` was added on top of `SLUG`. Now, it isn't. Is this intentional?

[Same for `TAG` and `AUTHOR`]

Reply:
Yes, this seems simpler to me, especially if for some reason you don't want some of the substitutions in `SLUG_SUBSTITUTIONS` to take effect for categories. Note that `AUTHOR` behaved differently previously, I also thought it simpler to bring it in line.

Reply:
Agreed. Just making sure it was intended :).

PS: Special handling of `AUTHOR` was annoying me as well.